### DISCUSSION TITLE
Wormhole Limit Increase Proposal for Sui Ecosystem

### DISCUSSION THREAD

**Discussion by cynthiaqqq** [wormhole-foundation/wormhole#3704] [Governance]
Sui DeFi is off to an ambitious start with TVL rising to over $230M and daily spot volume reaching $30-50M. This growth has taken place in the short-time frame of less than 8 months since Sui Mainnet launch, breaking various growth records. Indeed, to date Sui DeFi is the fastest growing non-EVM chain. 

While Sui DeFi is still in the experimentation phase and early era of product-market fit, a key driver behind this growth is the chain’s technology. Sui’s object-centric architecture, parallelizable execution, and move programmability ensure that DeFi protocols can harness the highest throughput, at the lowest latency in all of web3. Offering DeFi users the ability to trade with unparalleled performance and the ability to truly leverage the chain’s composability and interoperability.

![image6](https://github.com/wormhole-foundation/wormhole/assets/81078066/710abf5b-f4c0-47ea-84cc-ee508e5820b9)

Wormhole has been a key enabler of Sui DeFi, reaching strong synergy with the chain’s main dApps by bootstrapping their growth with bridged liquidity from older and more established chains. Indeed, in the last 90 days Sui has been both the main recipient and source of Ethereum volume, with 55.5% of outbound volume and 55.1% of inbound volume, respectively. These flows indicate an especially strong synergy with the Ethereum ecosystem, with Wormhole playing the role of key liquidity mediator.

![image1](https://github.com/wormhole-foundation/wormhole/assets/81078066/5cb71fae-182f-4022-ad45-c6a86781a5de)

![image4](https://github.com/wormhole-foundation/wormhole/assets/81078066/76ab2877-8a17-4293-ba03-7e6c06688416)

A similar story is true on the asset side. Sui plays a pivotal role in both of Wormhole’s top 2 assets, with Sui being the second most important chain for USDC volume and the most important chain for Tether volume. In USDC’s case, Sui accounts for 21% of the more than $500M of USDC volume bridged through Wormhole in the last 30 days. In the case of Tether, Sui accounts for 42% of the $175M of Tether volume bridged through Wormhole in the last 30 days. Both assets indicate the strong partnership between the Sui ecosystem and the Wormhole protocol.

![image2](https://github.com/wormhole-foundation/wormhole/assets/81078066/c18ba7f1-0ee0-421b-b91d-8539c7708e8c)

![image3](https://github.com/wormhole-foundation/wormhole/assets/81078066/20742e84-7560-484d-b2c0-03f5e4895590)

Critically, the limits on daily inflows and outflows into Sui remain fixed at their [initial levels](https://github.com/wormhole-foundation/wormhole/blob/main/node/pkg/governor/mainnet_chains.go) of $5M. In light of the above numbers, these limits are constraining the liquidity required for greater adoption of Sui DeFi and consolidating the role of Wormhole as the key liquidity bridge into the Sui ecosystem.
 
Indeed, as can be seen from the following chart, these limits have been hit 39 times in the last 90 days for inbound traffic and 26 times in the last 90 days for outbound traffic, indicating that they are indeed too severe relative to demand.

![image5](https://github.com/wormhole-foundation/wormhole/assets/81078066/69f0779c-3ff0-48b3-82e4-91b1e86e7ba2)

![image7](https://github.com/wormhole-foundation/wormhole/assets/81078066/b356d421-873c-453b-9b3d-2b3181926cfa)

To this end, the Sui Foundation is submitting a request that Wormhole increases its limits to $25M in order to usher in the next wave of Sui growth and cement its partnership with the Sui ecosystem.

**Comment by tonyjin** [comment]
One clarification - the Governor limit applies to outflows from a chain, so the current $5M Sui limit for small transactions does not limit inflows.

Inflows are limited by the respective outflow Governor limits of each chain going into Sui.

**Reply by elliscopef** [reply]
Yes, this is clearly communicated. The reason this proposal was made cause lots of 10m+ whales are concerned abt moving assets in due to the small cap on outflow and 5m outflow had been congested over the past few weeks

**Reply by elliscopef** [reply]
@tonyjin can we get this merge done soon?

**Comment by 1xCL** [comment]
I think we need to increase the limit ASAP.

**Comment by SimonKraus** [comment]
I quote the numbers from the discussion start:
"TVL rising to over $230M and daily spot volume reaching $30-50M"

The sheer fact that - as of today - the TVL bypassed $600M and the daily spot volume averaging between $50M and more than $100M is a clear indicator that the governor limits are long overdue for an increasement. 

Staking Facilities chiming in with an EASY AAYYYYEEEE!

**Comment by luca-git** [comment]
Increase the damn limit please, I'm facing issues every single day. The interaction sol sui is strong and getting stronger. Money is always stuck and ppl are forced to take absurd exchange rate on stables.

**Comment by nullbitx8** [comment]
+1 for increasing the limits.
 
Use case is cross-chain arbitrage with stables.
